apps/emqx_authz/src/emqx_authz.erl

```diff
@@ -204,26 +219,10 @@ check_dup_types([Source | Sources], Checked) ->
             check_dup_types(Sources, [Type | Checked])
     end.
 
-create_dry_run(T, Source) ->
-    case is_connector_source(T) of
-        true ->
-            [CheckedSource] = check_sources([Source]),
-            case T of
```

Comment:
This case clause seems not have been working.

Going to test this better with some real resource backends (without mecking `emqx_resource`) when they are introduced.